Source/ThirdParty/ANGLE/changes.diff

```diff
@@ -1,3 +1,139 @@
+diff --git a/.gitmodules b/.gitmodules
+index 6a5d4a9f2b638ddb9f3d59d01b37ccb4c76adcab..cc30569e4ac565aa5f77c42d8d029e9780fd785d 100644
```

Comment:
you need to copy the .gitmodules manuallly from the upstream revision you¨re merging and regenerate the diffs

Reply:
Thanks for the heads up.